### ISSUE TITLE
Mark Transcripts As Machine-Generated

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#5143]
# Description
Continuing from #5054, adding support for marking transcripts as machine-generated.

The checkbox being a per-transcript attribute that could be set or unset at any time. So an extra element on the page here, where you would also go to edit the transcript's label:
![Screenshot](https://user-images.githubusercontent.com/1406618/222804602-0bd47503-acd8-4b10-a1f3-5ec8070a36f9.png).

# Done Looks Like
- [x] Checkbox for noting whether a transcript is machine-generated or not is present when editing files on a section
- [x] If a file is specified as machine-generated, that checkbox should be checked by default when the user goes to edit the transcript file.

**Comment by masaball** [comment]
#5180 handles the Avalon specific work for this ticket. There is a checkbox when editing the transcript label that, when checked and saved, will add a tag to the transcript file marking it as machine generated. Current behavior also adds " (machine generated)" to the label for the file. The label is used as the filename on download, so this covers that requirement in the ticket.

Once it is merged, our next steps will be to update the transcript viewer to include the static message "Machine-generated transcript may contain errors" when a file has the `machine_generated` tag. Additionally, there may be some javascript adjustment necessary to retain parity with current edit page functionality, or adjustments may need to be made for how filenames are generated by the transcript viewer download. [See here for more detail](https://github.com/avalonmediasystem/avalon/pull/5180#discussion_r1181593859).

**Comment by elynema** [comment]
From discussion on the #av in iiif workspace, we talked with Mat Jordan at Northwestern about labeling transcripts as machine-generated.

Since it has been recommended that transcripts be `supplementing` annotations, he suggested: 

Could that tag be carried in the label of the annotation body?
"label": { "en": ["Transcript (Generated)"] }

Dananji noted that we could get in trouble if we are looking for the English words "Transcript (Generated)" to base functionality because the label could be in another language. From the viewer perspective, it might be ok to display any label associated with a supplementing AV annotation at the start of the transcript. We'd have to make the label i the IIIF manifest more of a note, like "Machine-generated transcript may contain errors."

On the download side, though, we'd either need a short label to add to the filename, or we need to actually cue off the text of the label.

Is there a way to auto-translate into English on the back-end so we can actually "read" the label? Maybe that could be future work if it is potentially feasible with some kind of auto-translate tool?

**Comment by masaball** [comment]
The merged PR ready for QA only adds a checkbox to the edit field and provides a tag in the database to designate items. 

Work left on this ticket is to add the 'Machine generated transcript may contain errors' to the transcript viewer in Ramp and to build out the Frontend display and Download within Ramp. Should these two steps be split into their own tickets so they don't get lost in the shuffle?
